src/styles/Navbar.module.css

```diff
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css2?family=Monoton&display=swap');
+
+.header {
+  background-color: var(--clr-white);
+  margin-bottom: 1rem;
+  border-bottom: solid 1px var(--clr-light-grey);
+}
+
+.nav {
+  gap: 2.5rem;
+}
+
+.logo {
+  font-family: 'Monoton', cursive;
+  font-size: 2em;
+  line-height: 1.4;
+  color: var(--clr-blue);
+  padding-bottom: 0.25em;
+}
```

Comment:
- Great Job For having this beautiful UI 👍👍 however you have to modify it to the given [zepline](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design as it is part of the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md) 👌
### here is your design
<img width="1146" alt="Screenshot 2023-03-03 at 12 12 07 PM" src="https://user-images.githubusercontent.com/52010789/222680015-e1b2c090-7a8d-4cc5-9a6b-008577bfc23e.png">

### here is [zepline](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design
![Screenshot 2023-03-03 at 12 02 18 PM](https://user-images.githubusercontent.com/52010789/222679538-da064e37-5a97-4180-b20b-39edf84e5dc3.png)

Reply:
Hi, @binhussen. Thanks for the feedback.

The design has features like the completion progress, chapters, Update progress functionality, comments, and edit. 

On this project, we do not have these features. We only have the book title, author, and the delete button. I cannot implement those other features when the project does not include them.

Please let me know how you think I should go forward.